### ISSUE TITLE
Grafana no data

### ISSUE THREAD

**Issue by nikossss** [grafana/grafana-zabbix#1477]
A few days ago , I updated the zabbix from 6.0 to 6.2
Update Grafana 7.9 to 9.02, update plugins to 4.2.9
At the same time , I reinstall the grafana and grafana-zabbix plugin , but when I create the panel , I got this error
By the way , my zabbix use Elasticsearch to store history data
json: invalid use of ,string struct tag, trying to unmarshal unquoted value into int64

**Comment by crosenbe** [comment]
Also no data available. I updated to zabbix 6.0, everything worked. Now, after some weeks no data are available anymore. 

Grafana: 9.0.4
Plugin: 4.2.9
Zabbix: 6.0.6

I removed the plugin, reinstalled it again. I created a new datasource, no success at all. No errors in the logs, what i can see in packet capture is the message 

`{"jsonrpc":"2.0","error":{"code":-32601,"message":"Method not found.","data":"Incorrect API \"application\"."},"id":2}`

Please notice after during the creation of the datasource, i got confirmed, API version 6.0.6 was detected.

**Comment by crosenbe** [comment]
More information of the not working request.

```
logger=plugin.alexanderzobnin-zabbix-datasource t=2022-07-24T10:48:06.036565297+02:00 level=debug msg="DS query" query="map[Interval:2e+09 JSON:map[application:map[filter:] datasource:map[type:alexanderzobnin-zabbix-datasource uid:sP-2c_g4z] datasourceId:2 functions:[] group:map[filter:] host:map[filter:/Fronius/] intervalMs:2000 item:map[filter:Direktverbrauch] itemTag:map[filter:] key:Q-0613df3e-da5a-42e3-960c-12a02b10cb0f-0 maxDataPoints:1537 options:map[disableDataAlignment:false showDisabledItems:false skipEmptyValues:false useZabbixValueMapping:false] proxy:map[filter:] queryType:0 refId:A resultFormat:time_series table:map[skipEmptyValues:false] tags:map[filter:] trigger:map[filter:] triggers:map[acknowledged:2 count:true minSeverity:3]] MaxDataPoints:1537 QueryType:0 RefID:A TimeRange:map[From:2022-07-24T09:48:07.256+02:00 To:2022-07-24T10:48:07.256+02:00]]"
logger=plugin.alexanderzobnin-zabbix-datasource t=2022-07-24T10:48:06.036710326+02:00 level=debug msg="Zabbix request" method=application.get
```

**Comment by crosenbe** [comment]
The method "application.get" is not available in the Zabbix documentation since 5.4. It seems it's finally gone.

**Comment by crosenbe** [comment]
I identified the root cause. Using only values in the query fields Host and Item will cause this issue. Using /*/ in the field Group additionally show data.
